karton/dashboard/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "1.2.0"
+__version__ = "1.2.1"
```

Comment:
1.2.0 is not yet released
```suggestion
__version__ = "1.2.0"
```